app/experimenter/kinto/tasks.py

```diff
@@ -151,7 +151,7 @@ def nimbus_push_experiment_to_kinto(collection, experiment_id):
         experiment.publish_status = NimbusExperiment.PublishStatus.WAITING
         experiment.save()
 
-        generate_nimbus_changelog(experiment, get_kinto_user())
+        generate_nimbus_changelog(experiment, get_kinto_user(), "Pushed to Kinto")
```

Comment:
Also in theory you could move these to constants on either NimbusConstants or the NimbusChangeLog model 🤷‍♀️

Reply:
Yup, [just did that in my latest update](https://github.com/mozilla/experimenter/pull/5130/files#diff-962a3f92cae878ce45f5b3dee466d34b97749882d62bf5ddcce0881482ead1f3R193-R199) after realizing I'd done it client-side but not server-side